distributions/distribution-resources/src/main/resources/etc/org.ops4j.pax.url.mvn.cfg

```diff
@@ -56,47 +56,28 @@ org.ops4j.pax.url.mvn.useFallbackRepositories=false
 
-#
-# Comma separated list of repositories scanned when resolving an artifact.
-# Those repositories will be checked before iterating through the
-#    below list of repositories and even before the local repository
-# A repository url can be appended with zero or more of the following flags:
-#    @snapshots  : the repository contains snaphots
-#    @noreleases : the repository does not contain any released artifacts
-#
-# The following property value will add the system folder as a repo.
-#
-org.ops4j.pax.url.mvn.defaultRepositories=\
-    file:${karaf.home}/${karaf.default.repository}@id=system.repository@snapshots
-#    file:${karaf.data}/kar@id=kar.repository@multi@snapshots, \
-#    file:${karaf.base}/${karaf.default.repository}@id=child.system.repository@snapshots
-
-# Use the default local repo (e.g.~/.m2/repository) as a "remote" repo
-#org.ops4j.pax.url.mvn.defaultLocalRepoAsRemote=false
 
 #
 # Comma separated list of repositories scanned when resolving an artifact.
-# The default list includes the following repositories:
-#    http://repo1.maven.org/maven2@id=central
-#    http://repository.springsource.com/maven/bundles/release@id=spring.ebr
-#    http://repository.springsource.com/maven/bundles/external@id=spring.ebr.external
-#    http://zodiac.springsource.com/maven/bundles/release@id=gemini
-#    http://repository.apache.org/content/groups/snapshots-group@id=apache@snapshots@noreleases
-#    https://oss.sonatype.org/content/repositories/snapshots@id=sonatype.snapshots.deploy@snapshots@noreleases
-#    https://oss.sonatype.org/content/repositories/ops4j-snapshots@id=ops4j.sonatype.snapshots.deploy@snapshots@noreleases
-# To add repositories to the default ones, prepend '+' to the list of repositories
-# to add.
+# 
 # A repository url can be appended with zero or more of the following flags:
 #    @snapshots  : the repository contains snapshots
 #    @noreleases : the repository does not contain any released artifacts
 #    @id=repository.id : the id for the repository, just like in the settings.xml this is optional but recommended
 #
-org.ops4j.pax.url.mvn.repositories=
-#    http://repo1.maven.org/maven2@id=central, \
-#    http://repository.springsource.com/maven/bundles/release@id=spring.ebr.release, \
-#    http://repository.springsource.com/maven/bundles/external@id=spring.ebr.external, \
-#    http://zodiac.springsource.com/maven/bundles/release@id=gemini, \
-#    http://repository.apache.org/content/groups/snapshots-group@id=apache@snapshots@noreleases, \
-#    https://oss.sonatype.org/content/repositories/snapshots@id=sonatype.snapshots.deploy@snapshots@noreleases, \
-#    https://oss.sonatype.org/content/repositories/ops4j-snapshots@id=ops4j.sonatype.snapshots.deploy@snapshots@noreleases, \
-#    http://repository.springsource.com/maven/bundles/external@id=spring-ebr-repository@snapshots@noreleases
+# This contains the openhab, smart home and default repositories.
+#
+org.ops4j.pax.url.mvn.repositories= \
```

Comment:
These repos now all also end up in the offline distro. We have specifically removed them because we noticed that they lead to a bad startup performance, since Karaf checks all those repos (and runs into timeouts if offline).

Also for the online distro I would prefer to have only "our" repos (eclipse, jcenter, jfrog) configured to make sure that we have under control, what is available to the runtime. All stuff from Karaf itself should imho be already contained in the runtime and there should be no need to access anything of that remotely.